lib/reporters/default.js

```diff
@@ -515,9 +515,6 @@ export default class Reporter {
 
 	writeFailure(event) {
 		this.lineWriter.writeLine(colors.title(this.prefixTitle(event.testFile, event.title)));
-		if (!this.writeLogs(event, true)) {
-			this.lineWriter.writeLine();
```

Comment:
Looking at the diff, I think we need this empty line.

Reply:
I have added the empty line. According to the existing logic, an empty line will be added when there are no logs.